src/manifest.json

```diff
@@ -22,11 +22,16 @@
   },
   "permissions": [
     "storage",
+    "identity",
     "https://ssl.google-analytics.com/collect"
   ],
   "background": {
     "scripts": [
+      "vendor/jose.js",
+      "vendor/kinto-http.js",
+      "vendor/fxa-crypto-relier/fxa-crypto-relier.js",
```

Comment:
See https://github.com/mozilla/notes/pull/141/files#r139488935 comment, re: do we need that extra `/fxa-crytpo-relier/` directory with 1 file?